src/main/java/roomescape/controller/TimeController.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.domain.Time;
+import roomescape.service.TimeService;
+
+import java.net.URI;
+import java.util.List;
+
+@Controller
+public class TimeController {
+    private final TimeService timeService;
+
+    @Autowired
+    public TimeController(TimeService timeService) {
+        this.timeService = timeService;
+    }
+
+    @GetMapping("/time")
+    public String showTimePage() {
+        return "time";
+    }
+
+    @PostMapping("/times")
+    @ResponseBody
+    public ResponseEntity<Time> createTime(@RequestBody Time time) {
+        if(!timeService.isValidTime(time)) {
+            return ResponseEntity.badRequest().build();
+        }
+
+        Time newTime = timeService.createTime(time);
+
+        return ResponseEntity.created(URI.create("/times/" + newTime.getId()))
+                .body(newTime);
+    }
+
+    @GetMapping("/times")
+    @ResponseBody
+    public ResponseEntity<List<Time>> getTimes() {
+        List<Time> times = timeService.getAllTimes();
+        return ResponseEntity.ok().body(times);
+    }
+
+    @DeleteMapping("/times/{id}")
+    @ResponseBody
+    public ResponseEntity<?> deleteTime(@PathVariable Long id) {
+        boolean isDeleted = timeService.deleteTime(id);
+        if (isDeleted) {
+            return ResponseEntity.noContent().build();
+        } else {
+            return ResponseEntity.badRequest().build();
+        }
```

Comment:
```suggestion
        boolean isDeleted = timeService.deleteTime(id);
        if (isDeleted) {
            return ResponseEntity.noContent().build();
        }
        return ResponseEntity.badRequest().build();
```
else문 대신 early return 방식을 사용하는 것이 더 좋습니다!


또, isDeleted가 false인 상황이 예외적인 상황이기 때문에 이를 강조하기 위해 아래와 같이 isDeleted가 false인지를 if문 넣어도 좋을 것 같아요!
```suggestion
        boolean isDeleted = timeService.deleteTime(id);
        if (!isDeleted) {
            return ResponseEntity.badRequest().build();
        } 
        return ResponseEntity.noContent().build();
```


또는 앞서 말했듯이 '예외'인 상황이기 때문에 서비스 레이어나 컨트롤러 레이어 중 적합한 위치에서 예외를 발생시키고 이 exception을 GlobalExceptionHandler에서 핸들링해주는 방식도 좋을 것 같습니다!

Reply:
네! if문 조건을 수정하고, early return으로 수정하였습니다. 
예외로 처리하는 방법도 참고하겠습니다!